### PR TITLE
Update variables.tf

### PR DESCRIPTION
**Pull request by WoodProgrammer**
Hi ,

When I use the EFS module, it give to me a error shown as below ;

     provisioned_throughput_in_mibps: cannot parse '' as float: strconv.ParseFloat: parsing "": invalid syntax

Error fixed when I changed the default variable of provisioned_throughput_in_mibps string to float .

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -72,7 +72,7 @@ variable "performance_mode" {
 }
 
 variable "provisioned_throughput_in_mibps" {
-  default     = ""
+  default     = 1.0
```
Comment (osterman) [inline variables.tf]: I wonder if we should try `0.0` instead. It's odd that TF (or AWS) is complaining when `throughput_mode` is defaulted to `bursting`.

Comment (aknysh) [inline variables.tf]:
```suggestion
  default     = 0
```

Comment (aknysh) [reply variables.tf]: i tested it with just `0` and it was not complaining.
@WoodProgrammer  can you please update the default, and then rebuild `README` by executing these commands:

```
make init
make readme/deps
make readme
```

thanks

**Comment by WoodProgrammer** [comment]
I've updated variable by your suggestion  . @aknysh

**Comment by aknysh** [comment]
thanks @WoodProgrammer 
can you also rebuild README by executing these commands in the module folder:

```
make init
make readme/deps
make readme
```

**Comment by aknysh** [comment]
thanks @WoodProgrammer

**Comment by WoodProgrammer** [comment]
you are welcome !

**Pull request merged**